conda_manager/widgets/main_window.py

```diff
@@ -116,23 +116,28 @@ def set_environments(self, prefix):
         """ """
         self.packages.set_environment(prefix=prefix)
 
-    def add_env(self):
+    @staticmethod
+    def add_env():
```

Comment:
These shouldn't be removed since I guess that `self` will be used in the methods. Did you use an automated program to clean the code ?